uavcan/register/384.Access.1.0.uavcan

```diff
@@ -64,19 +64,68 @@
 # The values of registers containing non-scalar numerical entities should be compared elementwise.
 #
 # The following table specifies the register name patterns that are reserved by the specification for
-# common functions. Implementers should always follow these conventions whenever applicable.
-# The table uses the following abbreviations for register flags:
-#   - M - mutable, I - immutable
-#   - P - persistent, V - volatile
-#
-#   Name pattern        | Flags | Type      | Purpose
-# ----------------------+-------+-----------+--------------------------------------------------------------------------
-#   uavcan.node_id      | MP    | uint16    | Contains the node-ID of the local node. Values above the maximum valid
-#                       |       |           | node-ID for the current transport indicate that the node-ID is not set;
-#                       |       |           | if plug-and-play is supported, it will be used by the node to obtain an
-#                       |       |           | automatic node-ID. Invalid values other than 65535 should be avoided for
-#                       |       |           | consistency. The factory-default value should be 65535.
-# ----------------------+-------+-----------+--------------------------------------------------------------------------
+# common functions. These conventions are not mandatory to follow, but implementers are recommended to adhere because
+# they enable enhanced introspection capabilities and simplify device configuration and diagnostics.
+#
+#   REGISTER NAME PATTERN                             TYPE            FLAGS                   RECOMMENDED DEFAULT
+# =====================================================================================================================
+#
+#   uavcan.node.id                                      natural16       mutable, persistent     65535 (unset/PnP)
+#
+# Contains the node-ID of the local node. Values above the maximum valid node-ID for the current transport
+# indicate that the node-ID is not set; if plug-and-play is supported, it will be used by the node to obtain an
+# automatic node-ID. Invalid values other than 65535 should be avoided for consistency.
+#
+# ---------------------------------------------------------------------------------------------------------------------
+#
+#   uavcan.node.description                             string          mutable, persistent     (empty)
+#
+# Human-readable verbose description of this specific node defined by the integrator (not vendor).
```

Comment:
```suggestion
# User-defined, human-readable description of this specific node. This is intended for use by a system integrator and should not be set by the manufacturer of a component. For example: on a quad-rotor drone this might read "motor 2" for one of the ESC nodes.
```

I'm not sure I'd call this verbose with only 256 bytes to work with (especially if you are using a lot of non-ascii characters).